coffee-backend/src/main/kotlin/chat/letscoffee/friends/FriendRequestController.kt

```diff
@@ -0,0 +1,50 @@
+package chat.letscoffee.friends
+
+import chat.letscoffee.exception.ResourceNotFoundException
+import chat.letscoffee.security.model.User
+import chat.letscoffee.security.repository.UserRepository
+import chat.letscoffee.security.security.CurrentUser
+import chat.letscoffee.security.security.UserPrincipal
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.*
+import javax.websocket.server.PathParam
+
+@RestController
+@RequestMapping("/friends/requests")
+class FriendRequestController(private val userRepository: UserRepository, private val service: FriendRequestService) {
+
+    data class FriendRequestPost(val to: String, val workaround: String?);
```

Comment:
Didn't spot this before now... What's a workaround? 🤔

Reply:
Apparently, there's a bug in Spring, that it cant automatically parse a Data class with only one variable..

Reply:
Then we can just use a String instead ;)